src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -183,6 +194,13 @@ async function processComponent( component: string, option: string ) {
 	return result;
 }
 
+export function resolvePath( input: string ): string {
+	// Resolve does not do ~ reliably
+	const resolvedPath = input.replace( /^~/, os.homedir() );
```

Comment:
Is this already handled by some built-in function?

Reply:
Doesn't seem to from what I have found online - https://stackoverflow.com/a/37112508/11875271

Like there are some tools probably, but it seems kinda simple to introduce a new dependency.